helm-chart/templates/deployment.yaml

```diff
@@ -0,0 +1,89 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: "{{ .Values.controlPlane.name }}-deploy"
+  namespace: "{{ .Values.namespace }}"
+  labels:
+    helm.sh/chart: "{{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}"
```

Comment:
Should be added on every kubernetes resources? 😁

Reply:
Yes, according to the Helm documentation, rhe helm.sh/chart label is recommended to identify the chart name and version associated with a resource managed by Helm. 😄 https://helm.sh/docs/chart_best_practices/labels/

Reply:
What I mean is we should set those labels on all resources (not just the deployment) but maybe it's propagated ?